server/middlewares/profile.middleware.js

```diff
@@ -0,0 +1,39 @@
+import model from '../models';
+import validations from '../helpers/validations';
+
+const { User } = model;
+
+const middleware = {
+  /**
+   * @description Check for the user
+   * @param {object} req Request object
+   * @param {object} res Response object
+   * @param {function} next calls the next function
+   * @returns {object} error for invalid id
+   * @returns {object} error for user not found
+   * @returns {function} next function
+   */
+  async checkIfUserExist(req, res, next) {
+    if (!validations.verifyUUID(req.params.id)) {
+      return res.status(400).json({
+        error: 'id not valid',
+      });
+    }
+
```

Comment:
This should be in the profile controller. You don't need this `checkIfUserExist` middleware